packages/web-react/src/components/FileUploader/useFileUploaderStyleProps.ts

```diff
@@ -102,6 +121,7 @@ export const useFileUploaderStyleProps = (props?: FileUploaderStyleProps): FileU
         image: fileUploaderAttachmentImageClass,
         slot: fileUploaderAttachmentSlotClass,
       },
+      imageCropStyles: hasCoords ? imageCropCSS : undefined,
```

Comment:
Please, try to add the key and value conditionally. If `hasCoords` is `true`, the key `imageCropStyles` will exist but the value will be `undefined`. I think it is better approach not to add key if there is no value.

Reply:
key removed if there are no coords